src/benchmarks/clo.c

```diff
@@ -38,14 +38,19 @@
 #include <errno.h>
 #include <assert.h>
 #include <sys/queue.h>
+#include <inttypes.h>
 
 #include "benchmark.h"
 #include "scenario.h"
 #include "clo_vec.h"
 #include "clo.h"
 
+#ifndef min
```

Comment:
So in fact this is a more complicated matter, unfortunately. For historical reasons, Windows has `min` and `max` macros and you can't normally use `std::max` and `std::min`. You would have to undef it first...